project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/vendor/aos/aos.esm.js

```
@@ -0,0 +1,610 @@
+
+var callback = function callback() {};
+
+function containsAOSNode(nodes) {
+  var i = void 0,
+      currentNode = void 0,
+      result = void 0;
+
+  for (i = 0; i < nodes.length; i += 1) {
+    currentNode = nodes[i];
+
+    if (currentNode.dataset && currentNode.dataset.aos) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)